### ISSUE TITLE
BrowserWindow options being ignored

### ISSUE THREAD

**Issue by sam0x17** [youchan/electron_opal#1]
When initializing a browser window via `Electron::BrowserWindow.new`, the options in the options hash are largely ignored. Consider the following:

```ruby
# main.rb
app.on('ready') do
  win = Electron::BrowserWindow.new('win', debug: true, width: 1200, height: 1200)
end
```

Testing this on 0c0d5fc9c0fba6e7710c69285427dcf894cbdbef yields a window with the default 800x600 `width` and `height` instead of 1200x1200 (I have also tried other width/height numbers, so the problem isn't that the numbers are too high or something like that). I have also tested `minWidth`, `minHeight`, `transparent`, `frame` and a few others, all of which are also ignored.

Oddly enough, the `debug` flag does work, but my guess is that this has something to do with this line https://github.com/youchan/electron_opal/blob/master/opal/electron/browser_window.rb#L8.

I'll keep investigating and see if I can figure out why this is happening.

*temporary workaround:*
Use backticks to initialize the window in pure JavaScript

**Comment by sam0x17** [comment]
For anyone who is curious, the actual code I am trying to get to work is this:

```ruby
# main.rb
WINDOW_PARAMS = {
  transparent: true,
  frame: false,
  toolbar: false,
  width: 800,
  height: 600,
  minWidth: 750,
  minHeight: 550,
  resizable: true,
  debug: true
}

app = Electron.app

app.on('window-all-closed') {  app.quit }

app.on('ready') do
  main_window = Electron::BrowserWindow.new('main_window', WINDOW_PARAMS)
  main_window.on('closed') { main_window = nil }
end
```

**Comment by youchan** [comment]
I fix it 2b9b1198674ba50dd5f1037e089a85c6bd1db7e2

and some information

- You should pass the `debug` flag separately from `WINDOW_PARAMS`.
  https://github.com/youchan/electron_opal/blob/master/lib/opal/electron/browser_window.rb#L7
- [opal](https://github.com/youchan/electron_opal/tree/master/opal) directory is unused now. I'm rewiting.
